zcash_client_sqlite/src/wallet.rs

```diff
@@ -2953,10 +3160,11 @@ pub(crate) fn store_decrypted_tx<P: consensus::Parameters>(
                         txout,
                         d_tx.mined_height(),
                         &address,
-                        account_uuid,
                         false,
                     )?;
 
+                    receiving_accounts.insert(account_id, key_scope);
```

Comment:
Here by comparison we've fetched `key_scope` via a lookup.

Reply:
That's because we didn't do the determination elsewhere, as we do when receiving shielded notes.